app/channel.go

```diff
@@ -2396,8 +2428,8 @@ func (a *App) MoveChannel(team *model.Team, channel *model.Channel, user *model.
 		return err
 	}
 
-	if appErr := a.Srv().Store.Channel().UpdateSidebarChannelCategoryOnMove(channel, team.Id); appErr != nil {
-		return appErr
+	if nErr := a.Srv().Store.Channel().UpdateSidebarChannelCategoryOnMove(channel, team.Id); nErr != nil {
+		return model.NewAppError("MoveChannel", "app.channel.sidebar_categories.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
This should be `nErr.Error()`. Also see: https://github.com/mattermost/mattermost-server/pull/15268. This is a common bug that happens during copy-paste. Just a small note for your future PRs :)

Reply:
Yep my bad :facepalm: sorry about that panic. I will keep my eyes on these cases